### ISSUE TITLE
New dialog for the pixel on GTK UI backend

### ISSUE THREAD

**Issue by rollerozxa** [Bithack/principia#164]
Using the alpha channel to simulate a fourth object property in the form of the wireless frequency it listens on is cursed, unhelpful and the source of many headaches (see #160).

I don't know if GTK allows this but being able to have a dialog box that's the GTK colour picker in a slot and then a custom slider (and/or input box for precise control) below that for the wireless frequency, without having to make our own colour picker widget would be nice.

**Comment by griffi-gh** [comment]
yeah the color picker can be used as it's own widget (without the dialog)
